portfolio/src/main/webapp/index.html

```diff
@@ -1,19 +1,106 @@
+                    education. I have four sisters, and unfortunately no brother. Cameroon is a bilingual country and
+                    the two official languages are English and French. I grew up sepaking both, but I also learned
+                    Arabic and Pidgin English (creole based English spoken in West Africa and Hawaii). Hence, I speak
+                    four languages in total, and I am currently learning German!
+                </p>
+            </div>
+            <div class="bio-text">
+                <h3 class="section-heading">Hobbies</h3>
+                <p>
+                    In my free time, I like to paint and draw. My drawings mostly depict abstact art scenes. I love
+                    charcoal and graphite and I primarily use that media for my drawings. People often say charcoal is
+                    messy, but I think if carefully used, it produces the most beautiful images.
+                </p>
+                <div class="art-preview">
+                    <img class="art-piece" alt="hand reaching" src="images/collin-grimm-art.jpeg">
+                    <img class="art-piece" alt="darwin watterson" src="images/darwin.jpeg">
+                    <img class="art-piece" alt="fallen angel" src="images/fallen.angel.jpg">
+                </div>
+            </div>
+            <div class="bio-text">
+                <h3 class="section-heading">Projects</h3>
+                <ul class="project-list">
+                    <li class="project-list-item">
+                        <h4 class="project-list-item-title"><a
+                                href="https://github.com/georgetown-cset/weak-labeling-experiments">Snorkel for Data
+                                Labeling</a></h4>
+                        <p>This project attempts to use Snorkel to
+                            create weak labels for large amounts of training data at Georgetown CSET using weak
+                            supervision. The
+                            data is primarily paper abstracts, whose relevance to various WIPO Application Areas we
+                            explore. This was my summer project at my last internship where I used weak supervision to
+                            label about 4GB worth of paper abstracts by writing weak supervison models based on Snorkel
+                        </p>
+                    </li>
+                    <li class="project-list-item">
+                        <h4 class="project-list-item-title"><a
+                                href="https://github.com/jsonkao/alfred-prufrock-editor">Free Write (Alfred Prufrock
+                                Editior)</a></h4>
+                        <p>This project uses Google's NLP APIs to create a mind map of someone's thought process, and
+                            builds a dynamic minndmap that shows both sentiment of thoughts and relationship to other
```

Comment:
nit: spelling mistake "mindmap"? It is spelled as "mind map". It would be good to be consistent.

Reply:
Corrected the typo. Thanks for the feedback